sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_custom_documents_async.py

```diff
@@ -34,6 +34,17 @@
 import asyncio
 
 
+def format_bounding_region(bounding_regions):
+    if not bounding_regions:
+        return "N/A"
```

Comment:
the user will see  `... within N/A bounding regions` right?
what about not printing that line at all if the bounding region is not present? 

Same comment will apply for bounding box

Reply:
Just a general comment on this - in order to keep the samples simple and focused in v2 we created a separate sample `get_bounding_boxes` for people who were more interested in the location of text on the page... I'm kind of inclined to do the same here so we can leave this sample to just be mostly fields/values/confidences

Reply:
That sounds good...I'll remove this from the sample

Reply:
Done